src/transformStateWithClones.js

```diff
@@ -8,6 +8,29 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const result = [];
+  const currentState = { ...state };
+
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
```

Comment:
Great job! you can also use switch case for better readability

Reply:
> Great job! you can also use switch case for better readability

Good point, if something doesn't match the checklist don't hesitate to reject